Common/UnitDefinitions/SpecificFuelConsumption.json

```diff
@@ -23,8 +23,8 @@
     {
       "SingularName": "KilogramPerKilogramForceHour",
       "PluralName": "KilogramsPerKilogramForceHour",
-      "FromUnitToBaseFunc": "{x} * 28.33",
-      "FromBaseToUnitFunc": "{x} / 28.33",
+      "FromUnitToBaseFunc": "{x} * 453.59237 / (0.0044482216152605 * 3600)",
+      "FromBaseToUnitFunc": "{x} * (0.0044482216152605 * 3600) / 453.59237",
```

Comment:
FromUnitToBaseFunc should be: `1000 / (9.80665e-3 * 3600)`
for base unit GramPerKilonewtonSecond

Reply:
Fixed in https://github.com/angularsen/UnitsNet/pull/1499/commits/a471451d4fc9e9df7a2a1a970dd9a933f9d77772